pkg/config/settings/runtime_setting_block_profile_rate.go

```diff
@@ -35,6 +36,7 @@ func (r RuntimeBlockProfileRate) Set(value interface{}) error {
 	}
 
 	profiling.SetBlockProfileRate(rate)
+	config.Datadog.Set("internal_profiling.block_profile_rate", rate)
```

Comment:
What is the reason to change the config? When you set something in `datadog.yaml` and later call `config.Datadog.Get("Something")` you expect to have the value from the config file.

Reply:
So that the current values are reflected in the runtime config dump returned by `agent config`. Also for consistency with other runtime settings, which do the same thing.